app/src/main/java/ch/epfl/sdp/drone3d/ui/mission/ItineraryCreateActivity.kt

```diff
@@ -216,19 +217,18 @@ class ItineraryCreateActivity : BaseMapActivity(), OnMapReadyCallback,
             mapboxMap.addOnMapClickListener(this)
 
             // Buttons
-            altitudeText.text = getString(R.string.altitude_text, flightHeight)
+            altitudeText.text = getString(R.string.altitude_text, flightHeight + MINIMUM_FLIGHTHEIGHT)
```

Comment:
I'm not sure to know exactly what fleightHeight represents, but I find a bit weird that we have to add MINIMUM_FLIGHTHEIGHT here and that it is not somehow implicit in flightHeight,

Reply:
The vertical seekbar used for the altitude return a progress between 0 and 100, so we need shift the progress value to correspond to  altitude. However you right it wasn't clear, so I improved it.